src/name/resolve.js

```diff
@@ -45,14 +45,14 @@ module.exports = (createCommon, options) => {
 
       const value = fixture.cid
 
-      ipfs.name.publish(value, (err, res) => {
+      ipfs.name.publish(value, { 'allow-offline': true }, (err, res) => {
         expect(err).to.not.exist()
         expect(res).to.exist()
 
         ipfs.name.resolve(nodeId, (err, res) => {
           expect(err).to.not.exist()
           expect(res).to.exist()
-          expect(res.path).to.equal(`/ipfs/${value}`)
+          expect(res).to.equal(`/ipfs/${value}`)
```

Comment:
Was it always failing before (when `.path` was expected), or is this change introduced by other PR?